share/shwap/sample_test.go

```diff
@@ -1,6 +1,7 @@
-package shwap
+package shwap_test
```

Comment:
It should stay shwap, no?

Reply:
It is necessary to resolve import cycle of testing pkg, that relies on eds for support functions.